apps/web/src/hooks/useFileOperations.ts

```
@@ -151,3 +151,102 @@ export function useUploadFile() {
+
+			if (!response.ok) {
+				const errorData = await response.json().catch(() => ({}));
+				throw new Error(errorData.message || "Failed to download file");
+			}
+
+			// Get the filename from the Content-Disposition header
+			const contentDisposition = response.headers.get("Content-Disposition");
+			let filename = fileName || "download";
+			if (contentDisposition) {
+				const filenameMatch = contentDisposition.match(/filename="(.+)"/);
+				if (filenameMatch && filenameMatch[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)